rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1692,9 +1726,9 @@ def test_upgrade_db_24_to_25(user_data_dir):  # pylint: disable=unused-argument
         assert len(raw_upgraded) == asset_movements_count - 2, 'coinbase/pro movements should have been deleted'  # noqa: E501
     # Check that identifiers match with what is expected. This may need amendment if the upgrade test stays around while the code changes.  # noqa: E501
     with db.conn.read_ctx() as cursor:
-        movements = db.get_asset_movements(
+        movements = get_asset_movements(
             cursor=cursor,
-            filter_query=AssetMovementsFilterQuery.make(),
+            filter_query=AssetMovementsFilterQuery.make(timestamp_attribute='time'),
```

Comment:
Don't even do that. Since the timestamp attribute is useless, just hard code the old query in the new function you made there.